src/Uno.UI/Configuration.cs

```diff
@@ -38,6 +38,19 @@ public static class FrameworkElement
 			/// is set to <see cref="false"/>.
 			/// </summary>
 			public static bool ClearPreviousOnStyleChange { get; set; } = true;
+
+#if __ANDROID__
```

Comment:
Might be just my taste but I'd prefer if the flag is available everywhere so I don't have to put #if __ANDROID__ in the applications that want to use it. The name is already pretty clear.

Reply:
I suppose for internal purposes it's better to make it unavailable, to make sure it's never used in shared code.  Refuting myself